workflows/rnaseq/main.nf

```diff
@@ -785,6 +785,105 @@ workflow RNASEQ {
     emit:
     multiqc_report = ch_multiqc_report // channel: /path/to/multiqc_report.html
     versions       = ch_versions       // channel: [ path(versions.yml) ]
+
+    publish:
+    QUANTIFY_STAR_SALMON.out                            >> 'star_salmon'
+    QUANTIFY_STAR_SALMON.out.multiqc                    >> null
+    QUANTIFY_STAR_SALMON.out.merged_counts_transcript   >> null
+    QUANTIFY_STAR_SALMON.out.merged_tpm_transcript      >> null
+    QUANTIFY_STAR_SALMON.out.versions                   >> null
```

Comment:
Here's an example of publishing all outputs of a module/subworkflow and disabling the ones you don't want